docs/deploy/03-microsoft-entra-id.md

```diff
@@ -32,7 +32,12 @@ This does not configure anything related to workload identity. This configuratio
    > :bulb: Skip the `az login` command if you plan to use your current user account's Microsoft Entra ID tenant for Kubernetes authorization. *Using the same tenant is common.*
 
    ```bash
-   az login -t <Replace-With-ClusterApi-AzureAD-TenantId> --allow-no-subscriptions
+   az login -t <Replace-With-ClusterApi-EntraID-TenantId> --allow-no-subscriptions
+   ```
+
+1. Query and save the Kubernetes cluster's tenant ID.
```

Comment:
FYI @ferantivero I split this into two steps because I always missed the split myself :)